src/main/app/.env

```diff
@@ -1,2 +1,2 @@
-REACT_APP_BACKEND_URL=https://nivio-demo.herokuapp.com
+REACT_APP_BACKEND_URL=http://localhost:8080
```

Comment:
sollten eine local und eine deploy env anlegen, ansonsten könnten wir uns das ganze kaputt machen :D

Reply:
German now? ;)
Oops, I have mistakenly committed that file.

Reply:
reverted